app.py

```diff
@@ -7,119 +7,129 @@
 from datetime import *
 
 
-
 app = Flask(__name__)
 app.secret_key = 'polder_dashboard'
 app.permanent_session_life = timedelta(minutes=10)
 
 
-app.config['SQLALCHEMY_DATABASE_URI'] ='sqlite:///site.db'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///site.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 db = SQLAlchemy(app)
 bcrypt = Bcrypt(app)
 
+app.app_context().push()
 
-
```

Comment:
Generally what people do for constants is to write their names in all caps, like `START_DATE`, so that it's easier to remember to not reassign them later.